spec/support/flipper.rb

```diff
@@ -0,0 +1,13 @@
+module Flipper
+  RSpec.configure do |config|
+    config.before(:each) do |example|
+      allow(Panoptes).to receive(:flipper).and_return(Flipper.new(Flipper::Adapters::Memory.new))
+      Panoptes.flipper["cellect"].enable
+      Panoptes.flipper["designator"].enable
+      Panoptes.flipper[:remove_complete_subjects].enable
```

Comment:
They're turned off manually in the "This feature is disabled" context blocks. They are always on for the purposes of every other spec.